To: vim_dev@googlegroups.com
Subject: Patch 7.3.120
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.120
Problem:    The message for an existing swap file is too long to fit in a 25
	    line terminal.
Solution:   Make the message shorter. (Chad Miller)
Files:	    src/memline.c


*** ../vim-7.3.119/src/memline.c	2011-01-17 20:08:03.000000000 +0100
--- src/memline.c	2011-02-13 14:18:08.000000000 +0100
***************
*** 4071,4079 ****
      }
      /* Some of these messages are long to allow translation to
       * other languages. */
!     MSG_PUTS(_("\n(1) Another program may be editing the same file.\n    If this is the case, be careful not to end up with two\n    different instances of the same file when making changes.\n"));
!     MSG_PUTS(_("    Quit, or continue with caution.\n"));
!     MSG_PUTS(_("\n(2) An edit session for this file crashed.\n"));
      MSG_PUTS(_("    If this is the case, use \":recover\" or \"vim -r "));
      msg_outtrans(buf->b_fname);
      MSG_PUTS(_("\"\n    to recover the changes (see \":help recovery\").\n"));
--- 4071,4079 ----
      }
      /* Some of these messages are long to allow translation to
       * other languages. */
!     MSG_PUTS(_("\n(1) Another program may be editing the same file.  If this is the case,\n    be careful not to end up with two different instances of the same\n    file when making changes."));
!     MSG_PUTS(_("  Quit, or continue with caution.\n"));
!     MSG_PUTS(_("(2) An edit session for this file crashed.\n"));
      MSG_PUTS(_("    If this is the case, use \":recover\" or \"vim -r "));
      msg_outtrans(buf->b_fname);
      MSG_PUTS(_("\"\n    to recover the changes (see \":help recovery\").\n"));
*** ../vim-7.3.119/src/version.c	2011-02-12 13:59:55.000000000 +0100
--- src/version.c	2011-02-15 11:56:14.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     120,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
257. Your "hundred-and-one" lists include well over 101 items, since you
     automatically interpret all numbers in hexadecimal notation.
     (hex 101 = decimal 257)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
